crates/bevy_app/src/hotpatch.rs

```diff
@@ -0,0 +1,34 @@
+use bevy_ecs::{event::EventWriter, HotPatched};
+use dioxus_devtools::{connect, subsecond::apply_patch, DevserverMsg};
+
+use crate::{Last, Plugin};
+
+/// Plugin connecting to Dioxus CLI to enable hot patching.
+#[derive(Default)]
+pub struct HotPatchPlugin;
+
+impl Plugin for HotPatchPlugin {
+    fn build(&self, app: &mut crate::App) {
+        let (sender, receiver) = crossbeam_channel::bounded::<HotPatched>(1);
+
+        connect(move |msg| {
```

Comment:
Could we get a quick comment saying what each of these two blocks do? It's not intuitive to me.

Reply:
added 👍